infrastructure/platform.tfvars

```diff
@@ -18,144 +18,15 @@ project_id = "<your project ID>"
-  autoscaling            = true
-  min_count              = 1
-  max_count              = 3
-  local_ssd_count        = 0
-  spot                   = false
-  disk_size_gb           = 100
-  disk_type              = "pd-standard"
-  image_type             = "COS_CONTAINERD"
-  enable_gcfs            = false
-  enable_gvnic           = false
-  logging_variant        = "DEFAULT"
-  auto_repair            = true
-  auto_upgrade           = true
-  create_service_account = true
-  preemptible            = false
-  initial_node_count     = 1
-  accelerator_count      = 0
-}]
-
-enable_gpu = true
-gpu_pools = [{
-  name                   = "gpu-pool"
-  machine_type           = "n1-standard-16"
-  node_locations         = "us-central1-b,us-central1-c"
-  autoscaling            = true
-  min_count              = 1
-  max_count              = 3
-  local_ssd_count        = 0
-  spot                   = false
-  disk_size_gb           = 100
-  disk_type              = "pd-standard"
-  image_type             = "COS_CONTAINERD"
-  enable_gcfs            = false
-  enable_gvnic           = false
-  logging_variant        = "DEFAULT"
-  auto_repair            = true
-  auto_upgrade           = true
-  create_service_account = true
-  preemptible            = false
-  initial_node_count     = 1
-  accelerator_count      = 2
-  accelerator_type       = "nvidia-tesla-t4"
-  gpu_driver_version     = "DEFAULT"
-}]
-
-enable_tpu = false
-tpu_pools = [{
-  name                   = "tpu-pool"
-  machine_type           = "ct4p-hightpu-4t"
-  node_locations         = "us-central1-b,us-central1-c"
-  autoscaling            = true
-  min_count              = 1
-  max_count              = 3
-  local_ssd_count        = 0
-  spot                   = false
-  disk_size_gb           = 100
-  disk_type              = "pd-standard"
-  image_type             = "COS_CONTAINERD"
-  enable_gcfs            = false
-  enable_gvnic           = false
-  logging_variant        = "DEFAULT"
-  auto_repair            = true
-  auto_upgrade           = true
-  create_service_account = true
-  preemptible            = false
-  initial_node_count     = 1
-  accelerator_count      = 2
-  accelerator_type       = "nvidia-tesla-t4"
-}]
-
-
-## pools config variables
-all_node_pools_oauth_scopes = [
-  "https://www.googleapis.com/auth/logging.write",
-  "https://www.googleapis.com/auth/monitoring",
-  "https://www.googleapis.com/auth/devstorage.read_only",
-  "https://www.googleapis.com/auth/trace.append",
-  "https://www.googleapis.com/auth/service.management.readonly",
-  "https://www.googleapis.com/auth/servicecontrol",
-]
-
-
-cluster_labels = {
-  "gke-profile" = "ai-on-gke"
-}
-
-all_node_pools_labels = {
-  "gke-profile" = "ai-on-gke"
-}
-
-all_node_pools_metadata = {
-  disable-legacy-endpoints = "true"
-}
-
-all_node_pools_tags = ["gke-node", "ai-on-gke"]
-
+create_network    = true
+network_name      = "ml-network"
+subnetwork_name   = "ml-subnet1"
+subnetwork_cidr   = "10.100.0.0/16"
```

Comment:
does this still have the limitation of only 1 cluster per network due to IP space?

Reply:
it can have more clusters as well in same network